webpack.config.js

```diff
@@ -1,54 +1,74 @@
-const path = require('path');
-const webpack = require('webpack');
+const path = require("path");
+const webpack = require("webpack");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 module.exports = {
+    mode: "development",
```

Comment:
There's a fair amount of duplication between this file and the externals config which could/should be deduplicated but that could be done later in a separate diff and it's blocking.

Reply:
Great ideas! I'll handle this in a followup- there's lots here already.